src/QueryPipeline/ExecutionSpeedLimits.h

```diff
@@ -2,13 +2,26 @@
 
 #include <Poco/Timespan.h>
 #include <base/types.h>
+#include <Common/ProfileEvents.h>
```

Comment:
Moving this to an include that ends up being in ~800 files seems unnecessary. `CurrentThread.h` is not used, and the 2 calls to `handleOverflowMode` could use basically the same string and arguments, so there is no need to put it into a header.

Reply:
This is still here